backend/apps/driving/models.py

```diff
@@ -0,0 +1,20 @@
+from django.core.validators import MaxLengthValidator
+from django.db import models
+
+
+class Chapter(models.Model):
+    text = models.TextField(validators=[MaxLengthValidator(2000)])
+
+    def __str__(self):
+        return self.text
+
+
+class Ticket(models.Model):
+    chapter = models.OneToOneField(to=Chapter, on_delete=models.CASCADE)
+    question = models.TextField(validators=[MaxLengthValidator(100)])
+    picture = models.ImageField()
```

Comment:
I'm not 100% sure if there is a sense to have it as `ImageField`. It has _a lot_ of peciliarities and may be an overkill for what you need.

For now, I guess you should stay with some ordinary `CharField` in which you'll keep a relative path to the image file or something like this.

Though, in case you'll decide to implement a possibility to _manage_ the images via admin — you'll have to go for `ImageField()` (but I'd recommend to keep images in a separate model, you'll see why after you'll try to keep it withing the ticket model for some time :) ).

So it's up to you.